website/api/blockchain-integration/contracts/verifier.md

```diff
@@ -38,28 +34,18 @@ In this example, the `IS_EVEN` zkVM program verifies that the number, `x`, is ev
 By verifying a receipt with the [image ID][term-image-id] of that program in a `require` statement, it is guaranteed that the stored number will always be even.
 RISC Zero's zkVM and the `IS_EVEN` program guarantee that it's computationally impossible to produce a verifying receipt for an odd number.
 
-## Versioning
```

Comment:
I feel like we should have some direct information about version management here, but we can revisit this with another PR.